apps/bfd-server/bfd-server-launcher/src/main/java/gov/cms/bfd/server/launcher/AppConfiguration.java

```diff
@@ -8,7 +8,13 @@
 
 /** Models the configuration options for the launcher. */
 public final class AppConfiguration implements Serializable {
-  private static final long serialVersionUID = -8134110339292440841L;
+  private static final long serialVersionUID = 1L;
```

Comment:
What's the reason for this change?

Reply:
Good question! Having computed serialization IDs provides a false sense of security, since nothing verifies them. (We also only use serialization for the test cases, so it doesn't matter, either way.)